com.github.matfantinel.reminduck.json

```diff
@@ -0,0 +1,72 @@
+{
+    "app-id": "com.github.matfantinel.reminduck",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.34",
+    "sdk": "org.gnome.Sdk",
+    "base": "io.elementary.BaseApp",
+    "base-version": "juno-19.08",
+    "command": "com.github.matfantinel.reminduck",
+    "finish-args": [
+       "--share=ipc",
+       "--share=network",
+       "--socket=x11",
+       "--socket=wayland",
+       "--talk-name=org.gtk.vfs.*",
+       "--talk-name=ca.desrt.dconf",
+       "--filesystem=home",
+       "--filesystem=xdg-run/dconf",
+       "--filesystem=~/.config/dconf:ro",
+       "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+    ],
+    "cleanup": [
+       "/include",
+       "/lib/pkgconfig",
+       "/man",
+       "/share/doc",
+       "/share/gtk-doc",
+       "/share/man",
+       "/share/pkgconfig",
+       "/share/vala",
+       "*.la",
+       "*.a"
+    ],
+    "modules": [
+       {
+          "name": "libyaml",
+          "sources": [
+              {
+                  "type": "git",
+                  "url": "https://github.com/yaml/libyaml.git",
+                  "tag": "0.2.1"
+              }
+          ]
+       },
+       {
+          "name": "appstream",
+          "buildsystem": "meson",
+          "config-opts": [
+              "-Dstemming=false"
+          ],
+          "sources": [
+              {
+                  "type": "git",
+                  "url": "https://github.com/ximion/appstream.git",
+                  "tag": "APPSTREAM_0_12_4"
+              }
+          ]
+       },
+       {
+          "name": "reminduck",
+          "buildsystem": "meson",
+          "sources": [
+              {
+                  "type": "git",
+                  "url": "https://github.com/matfantinel/reminduck.git",
```

Comment:
You probably want to patch the application to use the elementary GTK & icons themes. They are already in the BaseApp.

Reply:
I've tested it with many different themes in a Gnome environment and am happy with the result. I prefer it to respect GTK theming!